src/DistributedMetadata/DDLService.cpp

```diff
@@ -533,38 +562,38 @@ void DDLService::processRecords(const IDistributedWriteAheadLog::RecordPtrs & re
 {
     for (auto & record : records)
     {
-        if (record->op_code == IDistributedWriteAheadLog::OpCode::CREATE_TABLE)
+        switch (record->op_code)
         {
-            createTable(record);
-        }
-        else if (record->op_code == IDistributedWriteAheadLog::OpCode::DELETE_TABLE)
-        {
-            mutateTable(record, Poco::Net::HTTPRequest::HTTP_DELETE);
-
-            /// Delete DWAL
-            String database = record->block.getByName("database").column->getDataAt(0).toString();
-            String table = record->block.getByName("table").column->getDataAt(0).toString();
-            std::any ctx{DistributedWriteAheadLogKafkaContext{database + "." + table}};
-            doDeleteDWal(ctx);
-        }
-        else if (record->op_code == IDistributedWriteAheadLog::OpCode::ALTER_TABLE)
-        {
-            mutateTable(record, Poco::Net::HTTPRequest::HTTP_PATCH);
-        }
-        else if (record->op_code == IDistributedWriteAheadLog::OpCode::CREATE_DATABASE)
-        {
-            mutateDatabase(record, Poco::Net::HTTPRequest::HTTP_POST);
-        }
-        else if (record->op_code == IDistributedWriteAheadLog::OpCode::DELETE_DATABASE)
-        {
-            mutateDatabase(record, Poco::Net::HTTPRequest::HTTP_DELETE);
-
-            /// FIXME : Clean up tables DWAL in the database
-        }
-        else
-        {
-            assert(0);
-            LOG_ERROR(log, "Unknown operation={}", static_cast<Int32>(record->op_code));
+            case IDistributedWriteAheadLog::OpCode::CREATE_TABLE: {
```

Comment:
curly braces shall be kept in a separate line ?

Reply:
fixed